packages/screens/ApplicationRewiew/component/TeamInformation.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
```

Comment:
I saw that you files `BlablaInformation.tsx` are using the same structure of components, can you check if you can have a common components for these files ?

Reply:
yes, i checked before

Reply:
And it's not possible to share something between them ?

Reply:
inside this type of file have one common component  for same structure Box we use this  <BoxDetailTeriroy /> component and for structure Box we use this  <LinkCard />

![Screenshot 2024-01-23 at 4 21 30 PM](https://github.com/TERITORI/teritori-dapp/assets/154412187/8e6d6a24-c932-41ac-b257-86f2fe2fe124)

Reply:
BlablaInformation.tsx, this is for only reduce code and easy to read.